public/js/user-actions.js

```diff
@@ -27,6 +27,7 @@ module.exports = assign({}, {
       console.log('login succeeded');
       this.setCurrentUser({
         email: data.buyer_email,
+        payment_methods: data.payment_methods,
```

Comment:
jquery isn't very necessary in this day and age. i'd recommend using a specific request library if you need it for requests. we're using superagent (with a promise wrapper)